components/automate-ui/src/app/pages/user-details/user-details.component.ts

```diff
@@ -128,12 +129,16 @@ export class UserDetailsComponent implements OnInit, OnDestroy {
   }
 
   private createForms(fb: FormBuilder): void {
+    // Must stay in sync with error checks in user-details.component.html
     this.editForm = fb.group({
-      fullName: ['', Validators.required]
+      fullName: ['', [Validators.required, Validators.pattern(Regex.patterns.NON_BLANK)]]
```

Comment:
I thought we had talked about this -- would be it a bad idea to put `[Validators.required, Validators.pattern(Regex.patterns.NON_BLANK)]` into the constants collection?